### ISSUE TITLE
Highlighted text is hard to read on selection in the outline/breadcrumb views

### ISSUE THREAD

**Issue by misolori** [Microsoft/vscode#55952]
Because both colors are blue they become hard to read and fail the color contrast ratio. Would expect for the highlighted text in these selected states to become white.

Outline

| Current | Expected |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/35271042/43786658-04709832-9a1e-11e8-9b9f-1193624885b5.png)  | ![image](https://user-images.githubusercontent.com/35271042/43786668-0957e67a-9a1e-11e8-800b-12c22a0b47c5.png) |

Breadcrumb

| Current | Expected |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/35271042/43786908-a4591d06-9a1e-11e8-9de3-877922fa11bb.png) | ![image](https://user-images.githubusercontent.com/35271042/43787916-00fac8d2-9a21-11e8-8b04-6e81bc13c0ba.png) |

cc @jrieken @bpasero

**Comment by bpasero** [comment]
@misolori can you also attach the look for quick open after this change? and would this change be for the light theme only or all themes?

**Comment by misolori** [comment]
@bpasero I can't find an example that uses selection for quick open, do you know of one? And this change would be the new default for the breadcrumb/outline view, so it would affect all themes that use the "light theme" defaults in those two views. It's currently broken and was hard to see in our previous color as well:

![image](https://user-images.githubusercontent.com/35271042/43789162-2a767a28-9a24-11e8-839f-249886515b08.png)

**Comment by bpasero** [comment]
@misolori yeah, none of the pickers use selection color, I was more referring to how quick open would look like if you change highlight color to white. My understanding is that this would have an impact pretty much everywhere where highlighting is being used (independent from the tree or not)

**Comment by bpasero** [comment]
> so it would affect all themes that use the "light theme"

To clarify: this would affect all light themes, even the ones in the market place? or only our light theme

**Comment by misolori** [comment]
@bpasero sorry if I wasn't clear, my intention is to only change the highlight color when it's on a selection row. So I'm not intending to change the default highlight color. In the example below, the highlight color remains the same _except_ when it's selected:

![image](https://user-images.githubusercontent.com/35271042/43790172-aa5861b4-9a26-11e8-83fd-9e7450e007cf.png)

I still need to investigate how to do this, but my suspicion is that we already make the text lighter on a selected row, so I'd explore doing something like `color: inherit !important` while it's selected.

> To clarify: this would affect all light themes, even the ones in the market place? or only our light theme

yes, this would affect all light themes.

**Comment by bpasero** [comment]
@misolori but there is only `list.highlightForeground`, we do not distinguish the highlight color based on the type of selection currently.

**Comment by misolori** [comment]
@bpasero hmmm...let me do some investigation to figure this one out.

**Comment by jrieken** [comment]
>  we do not distinguish the highlight color based on the type of selection currently.

I think the point is the we don't have (and want) a highlighted-and-selected-foregournd-color but that the selection color always wins.

**Comment by misolori** [comment]
The highlight color on the dark theme also fails the color contrast ratio (3.2:1):

![image](https://user-images.githubusercontent.com/35271042/43849971-8870f866-9aeb-11e8-8f10-250bc044ffce.png)
